fineract-investor/src/main/java/org/apache/fineract/investor/service/ExternalAssetOwnersReadServiceImpl.java

```diff
@@ -48,4 +52,16 @@ public List<ExternalTransferData> retrieveTransferData(Long loanId, String exter
         }
         return result.stream().map(mapper::mapTransfer).toList();
     }
+
+    @Override
```

Comment:
Let's not let entities past through the transactional boundary (i.e. the service is annotated with Transactional so anything that's coming out of that will be detached) and could cause maintenance issues in the future (just like for the majority of fineract today).